models/profiles-ml.yaml

```diff
@@ -46,17 +42,14 @@ models:
               features:
                 - name: *percentile_name_7_days
                   description: 'Percentile of churn score. Higher the percentile, higher the probability of churn'
-
-      <<: *feature_meta_data_7_days
-
+      !!merge <<: *feature_meta_data_7_days
```

Comment:
Where is `!!merge` coming from ?

Reply:
@sprksh can you please check?

Reply:
This came via auto migrate. @shekhar-rudder

Reply:
https://blogs.perl.org/users/tinita/2019/05/reusing-data-with-yaml-anchors-aliases-and-merge-keys.html
So  `<<: ` is the shorthand for `!!merge <<:`
So while yaml migration, its converting to the explicit form.